src/collections/blog/2023/2023-05-22-changing-release-channels-in-meshery/index.mdx

```diff
@@ -0,0 +1,177 @@
+  mesheryctl or by editing your meshconfig file. In this blog post, we'll cover
+  both methods.
+</p>
+
+<br />
+<h3>
+  What is Meshconfig?
+</h3>
+<p>
+  Meshconfig is a configuration file that is used to configure Meshery. It is
+  typically located in the <code>~/.meshery</code> directory.It contains
+  information about the current release channel, the version of Meshery that is
+  installed, and other configuration options that are specific to your Meshery
+  installation. It is a YAML file, so you can edit it using any text editor.
+
+  Meshconfig is automatically generated when you run Meshery for the first time.
+  It is also automatically updated when you update Meshery. Meshconfig is a YAML
+  file, so you can edit it using any text editor. 
+</p>
+<h3>
+  Switching between Meshery release channels by editing your meshconfig file.
+</h3>
+
+<br />
+Another way to switch between Meshery release channels is by editing your config.yaml file. Here's how:
+
+<ol>
+<li>
+  Open your config.yaml file in a text editor.
+</li>
+<li>
+  Find the channel field in the meshery section of the file.
+</li>
+<li>
+  Change the value of the version field to the desired release channel, stable
+  or edge.
+</li>
+<li>
+  Save the changes to the file.
+</li>
+
+<li>
+  To confirm that the channel has been changed, run the following command again:
+
+<pre>
+  <code class="language-bash">mesheryctl system channel view</code>
+</pre>
+</li>
+</ol>
+
+<p>
+  After making these changes, future updates to Meshery will use the selected
+  release channel.
+</p>
+
+<br />
+<h3>Switching between Meshery release channels using mesheryctl.</h3>
+<br />
+
+<p>
+  mesheryctl is a command-line tool for managing Meshery. You can use it to switch between different release channels. Here's how:</p>
+<p>
+  Open your terminal and confirm that you have mesheryctl installed by running  <code>mesheryctl version</code>. If you don't have mesheryctl installed, you can install it by following the instructions in the  <a href="https://docs.meshery.io/installation/mesheryctl">Meshery documentation</a>.
+</p>
+
+<ul>
+<li>
+  Run the following command to see the current configuration for
+  Meshery:
+</li>
+
+<pre>
+  <code class="language-bash">mesheryctl system context view</code>
+</pre>
+
+<img src={MesheryVersion} className="image-center-no-shadow"  style="width:50%;"></img>
+
+<p>
+  This will show you the currently channels ,<b>stable</b> or <b>edge</b>, along with the version number and other information.
+</p>
+
+<li>Run the following command to switch to a different release channel:
+<pre><code class="language-bash">mesheryctl system channel set [stable|stable-version|edge|edge-version]
+</code></pre>
+  This command will update your local Meshery configuration to use the selected
+  channel for future updates. To set the channel to a specific version, replace <b>Version</b> with the desired version number.
```

Comment:
You need to specify how version number is determined. Also, usage of `mesheryctl system switch` is missing